p2p_wallet/Scenes/Main/Swap/Providers/JupiterTokensRepository.swift

```diff
@@ -81,7 +81,7 @@ final class JupiterTokensRepositoryImpl: JupiterTokensRepository {
                 localProvider.clear()
```

Comment:
Lets map isTokenCellVisibleOnWS on JupiterTokensRepository, not in ChooseSwapTokenService